### ISSUE TITLE
Allow push task to use a specific rubygems API key

### ISSUE THREAD

**Issue by joekhoobyar** [postmodern/rubygems-tasks#14]
This is analogous to the --key option to the gem push command:

```
$ gem help push
Usage: gem push GEM [options]

  Options:
    -k, --key KEYNAME                Use the given API key
                                     from ~/.gem/credentials
        --host HOST                  Push to another gemcutter-compatible host

```

**Comment by postmodern** [comment]
We could expose it as a configuration option in `Gem::Tasks::Push` and pass it into the `gem push` command.

**Comment by postmodern** [comment]
Implemented in the 0.3.0 branch.

**Comment by chadxz** [comment]
Hi @postmodern, I would love to use this feature. Is there anything currently blocking 0.3.0 release to rubygems that I might be able to help out with?
